src/components/ListingDetails/styles.ts

```diff
@@ -10,27 +10,11 @@ export const CaseDisplay = styled.aside`
 export const CaseTitle = styled(H1)`
   margin: 0.4rem 4rem;
 `;
```

Comment:
while we're modifying this file, let's go ahead and switch to pixels as well!

Reply:
sg! i changed all the other gap/margins to px as well!